doc/api/http.md

```diff
@@ -556,7 +556,7 @@ Returns `request`.
 added: v0.1.17
 -->
 
-This class inherits from [`net.Server`][] and has the following additional events:
+This�class inherits from [`net.Server`][] and has the following additional events:
```

Comment:
There's a Unicode white space there, that I converted to `\u0020`. I'll make sure that it's right